windows/jitsi-meet/render.js

```diff
@@ -1,23 +1,32 @@
+/* eslint-disable */
 /* global process */
 const remoteControl = require("../../modules/remotecontrol");
 let postis = require("postis");
 const setupScreenSharingForWindow = require("../../modules/screensharing");
 const config = require("../../config.js");
 const {dialog} = require('electron').remote;
+const ipcRenderer = require('electron').ipcRenderer;
+const WindowPeerConnection = require("../../modules/micromodeconnection").WindowPeerConnection;
+// const WindowPeerConnection = require("electron-peer-connection").WindowPeerConnection;
 
 /**
  * The postis channel.
  */
 let channel;
 
 /**
- * Cteates the iframe that will load Jitsi Meet.
+ * Cteates the jitsimeetiframe that will load Jitsi Meet.
  */
-let iframe = document.createElement('iframe');
-iframe.src = process.env.JITSI_MEET_URL || config.jitsiMeetURL;
-iframe.allowFullscreen = true;
-iframe.onload = onload;
-document.body.appendChild(iframe);
+ var options = {
+     domain: config.jitsiMeetURL,
+     room: "testurl",
+     width: '100%',
+     height: '100%'
```

Comment:
It seems like width and height options doesn't work for jitsi-meet ExternalAPI. Regardless of which number I give, it always creates an iframe of MIN width and height, 790 * 300.